assignments/assignment-5.adoc

```diff
@@ -441,7 +439,7 @@ The auto scaling group should have the following properties:
 
   * Group name: wordpress-web-group
   * Group size: 2
-  * Network: productionVpc
```

Comment:
The name of the VPC resource should be productionVpc. That's what my testing application expects and that's how it is noted in the diagram. Do you not like the resource name? :)

Reply:
Sounds good. I'm agnostic as to the casing. 

I noted, though, that 4 of the 7 occurrences of `productionV(pc|PC)` in the document included a capitalized `PC`. Three of these occurrences are highlighted in the screenshot below:

- ![image](https://user-images.githubusercontent.com/25967729/66709879-67f88a00-ed32-11e9-84fb-00657cc6e1f6.png)

If the automated tests are looking for `productionVpc`, perhaps the other entries should be changed.

Reply:
Sure, that makes sense.

Reply:
I'll adjust the edits for a consistent lowercase `pc`.

Though I suppose I do actually have a slightly vested interest in `productionVPC` since that's how I submitted my assignment. 😅